chcemvediet/tests/__init__.py

```diff
@@ -35,9 +36,14 @@ def setup_test_environment(self, **kwargs):
         super(CustomTestRunner, self).setup_test_environment(**kwargs)
         settings.LANGUAGE_CODE = u'en'
         os.environ[u'RECAPTCHA_TESTING'] = u'True'
+        self.settings_override = override_settings(
+            PASSWORD_HASHERS=(u'django.contrib.auth.hashers.MD5PasswordHasher',),
```

Comment:
Pls pridaj to docstringu hore na zaciatku `CustomTestRunner` odrazku:
```
 -- Used MD5 password hasher, it's much faster then the production hasher
```

Reply:
V runneri treba pouzivat `settings_override`? Nestaci priamo zapisat do `settings.PASSWORD_HASHERS`, ako to robime so `settings.LANGUAGE_CODE` o riadok vyssie?

Reply:
:ok:
Odstanil som zo `chcemvediet` app,  `AccountsTestCaseMixin` a tiez som odstranil v jednotlivych `setUp`
```
self.settings_override = override_settings(
            PASSWORD_HASHERS=(u'django.contrib.auth.hashers.MD5PasswordHasher',),
            )
        self.settings_override.enable()
```

. `setings_override` na zmenu hashera sa pozuiva aj v `poleno` app, vymazal som to aj odtial. Sice poleno je reusable app a nemala by byt zavisla na chcemvediet `CustomTestRunner`, ale testy zbehnu aj s pouzitim ineho hashera ako s MD5. Ak s tym nesuhlasis, mozem to vratit naspat.

Reply:
@viliambalaz Asi moze byt. Uplne po spravnosti by sme asi mali `CustomTestRunner` rozdelit na dva. V `poleno/utils/test.py` by sme mali definovat verziu `CustomTestRunner` v ktorej su vseobecne overridy platne pre testy v `poleno/` a potom v `chcemvediet/tests/__init__.py` by sme mali tento `CustomTestRunner` doplnit o veci, co su specificke pre `chcemvediet/` testy. Ale asi to nemusime riesit.